CMakeLists.txt

```diff
@@ -8,7 +8,7 @@
 
 cmake_minimum_required(VERSION 3.19 FATAL_ERROR)
 
-project(larg4 VERSION 09.08.00 LANGUAGES CXX)
+project(larg4 VERSION 09.09.00 LANGUAGES CXX)
```

Comment:
Revert this change as well--Lynn will make it when she's ready to tag the package.

Reply:
Note that updating a package version number in a PR may have unintended consequences when running the CI.